.meta/sinks/sematext.toml

```diff
@@ -0,0 +1,32 @@
+[sinks.sematext]
+title = "Sematext"
+batch_max_size = 10490000
+batch_timeout_secs = 1
+beta = true
+buffer = true
+common = true
```

Comment:
Should be `false`

Reply:
:+1: sounds good, are there docs on what each of these config items mean? I can't seem to find any.